dockers/docker-database/supervisord.conf.j2

```diff
@@ -56,3 +56,19 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
+
+{% if INSTANCES %}
+{%     for redis_inst, redis_items in INSTANCES.items() %}
+{% if redis_inst == 'redis_chassis' %}
```

Comment:
>if [](http://example.com/codeflow?start=3&length=2)

Need 2 level indentation

Reply:
Thanks for the review @qiluo-msft !
I will fix this.

Reply:
fixed.